test/e2e/operator_condition_e2e_test.go

```diff
@@ -134,34 +117,39 @@ var _ = Describe("Operator Condition", func() {
 
 		// Get the OperatorCondition for csvA and report that it is upgradeable, unblocking csvB
 		upgradeableTrueCondition := metav1.Condition{
-			Type:               operatorsv1.Upgradeable,
+			Type:               operatorsv2.Upgradeable,
 			Status:             metav1.ConditionTrue,
 			Reason:             "test",
 			Message:            "test",
 			LastTransitionTime: metav1.Now(),
 		}
 		Eventually(func() error {
-			cond, err = crc.OperatorsV1().OperatorConditions(testNamespace).Get(context.TODO(), csvA.GetName(), metav1.GetOptions{})
-			if err != nil {
+			cond, err = crc.OperatorsV2().OperatorConditions(testNamespace).Get(context.TODO(), csvA.GetName(), metav1.GetOptions{})
+			if err != nil || currentGen == cond.ObjectMeta.GetGeneration() {
```

Comment:
Is this just to check that the last update worked? It seems like we don't need this given that the spec was successfully earlier.

Reply:
This is simply to ensure all of reconciliation of OperatorCondition is completed and we get the latest version. This is to reduce flakes that tend to happen with e2e with all processes running in parallel.

Reply:
Typically I would go for a longer timeout, but this is fine.